be/src/olap/row_cursor.h

```diff
@@ -160,28 +115,44 @@ class RowCursor {
     inline size_t get_fixed_len() const { return _fixed_len; }
     inline size_t get_variable_len() const { return _variable_len; }
 
+    // Get column nullable pointer with column id
+    // TODO(zc): make this return const char*
+    char* nullable_cell_ptr(uint32_t cid) const {
+        return _fixed_buf + _schema->column_offset(cid);
+    }
+    char* cell_ptr(uint32_t cid) const {
```

Comment:
cell_ptr() in RowCursor not include null indicator, in ColumnBlock include null indicator.
The two place are not consistent.

Reply:
No, cell_ptr in ColumnBlock means doesn't include null indicator. In ColumnBlock, null indicator locates in a bitmap, cell_ptr only return field pointer